### ISSUE TITLE
Parameter name "body" and other form parameters.

### ISSUE THREAD

**Issue by caiiiycuk** [swagger-api/swagger-ui#325]
I have api POST method that takes few from parameters:

``` json
{
    "path":"/api/articles",
    "operations":[{
      "httpMethod":"POST",
      "summary":"Create a new article",
      "notes":"",
      "nickname":"ApiArticlesCreate",
      "parameters":[{
        "name":"apiKey",
        "paramType":"query",
        "type":"string",
        "description":"Use: 123",
        "required":true
      },{
        "name":"title",
        "paramType":"form",
        "type":"string",
        "description":"",
        "required":true
      },{
        "name":"body",
        "paramType":"form",
        "type":"string",
        "description":"",
        "required":true
      }],
      "responseMessages":[{
        "code":200,
        "message":"ID of the newly created article will be returned"
      }]
    }]
  }
```

But if one of parameter have name "body" than swagger-ui will send to server in POST body only value of this parameter and ignores others. But if i change name to "aBody" all works fine. In swagger.js:

``` javascript
  SwaggerRequest = (function() {

    function SwaggerRequest(type, url, params, opts, successCallback, errorCallback, operation, execution) {
//...
      body = params.body;
//...
```

It is normal?

Please see:
https://github.com/ngocdaothanh/xitrum/issues/181#issuecomment-25514938

**Comment by dylanninin** [comment]
Sad, it's been not fixed yet at [v2.1.8-M1](https://github.com/swagger-api/swagger-ui/tree/v2.1.8-M1)

**Comment by webron** [comment]
2.1.8-M1 is extremely old. Try using the latest.
